docs/tutorial/part-five/index.md

```diff
@@ -75,42 +75,42 @@ module.exports = {
 }
 ```
 
-Save that and restart the gatsby development server. Then open up GraphiQL again.
+Zapisz plik i uruchom serwer deweloperski Gatsby. Następnie otwórz ponownie GraphiQL.
 
-In the explorer pane, you'll see `allFile` and `file` available as selections:
+W panelu Explorer, zobaczysz `allFile` oraz `file` jako dostępny wybór:
 
 ![graphiql-filesystem](graphiql-filesystem.png)
 
-Click the `allFile` dropdown. Position your cursor after `allFile` in the query area, and then type <kbd>Ctrl + Enter</kbd>. This will pre-fill a query for the `id` of each file. Press "Play" to run the query:
+Kliknij w menu `allFile`. Ustaw kursor zaraz za `allFile` w obszarze zapytania, a następnie wciśnij <kbd>Ctrl + Enter</kbd>. To sprawi autouzupełnienie zapytania o `id` każdego z plików. Wciśnij "Play" aby wykonać zapytanie:
 
 ![filesystem-query](filesystem-query.png)
 
-In the Explorer pane, the `id` field has automatically been selected. Make selections for more fields by checking the field's corresponding checkbox. Press "Play" to run the query again, with the new fields:
+W panelu Explore, pole `id` zostało automatycznie zaznaczone. Wybierz więcej kategorii pól klikając w odpowiadające im pola wyboru. Wciśnij "Play" aby wykonać zapytanie, ponownie, tym razem z nowymi polami:
 
 ![filesystem-explorer-options](filesystem-explorer-options.png)
 
-Alternatively, you can add fields by using the autocomplete shortcut (<kbd>Ctrl + Space</kbd>). This will show queryable fields on the `File` nodes.
+Alternatywnie, możesz dodawać pola używając skrótu autouzupełnienia (<kbd>Ctrl + Space</kbd>). To pokaże wszystkie dostępne pola dla zapytań na Węzłach (nodes) `File`.
```

Comment:
Te węzły jakos nie pasują, ale szczerze mówiąc, nie mam za bardzo żadnej alternatywy. @lekterable @matborowiak  jakiś pomysł?

Reply:
j.w. uważam że pewnych terminów nie da się tłumaczyć dosłownie, i powinniśmy korzystać z "zapożyczeń" z angielskiego. Nie popadajmy w paranoje, skoro sami na codzien w języku polskim używamy słów jak "pluginy" "string" "array" "nody" itp. Jak będziemy tłumaczyć wszystko dosłownie to będzie to wyglądało nie dość że komicznie to jeszcze będzie nie do zrozumienia, bo te polskie słowa praktycznie nigdy w mowie codziennej nie występują w takim kontekście, a precież chodzi o to żeby dokumentacja była zrozumiała dla nas, Polaków XXI wieku. Takie moje zdanie. Co sądzicie?

Reply:
Zgadzam się jak najbardziej. Node (nie jakos node.js tylko ogólnie) zostawiłbym jako node.

Reply:
> Zgadzam się jak najbardziej. Node (nie jakos node.js tylko ogólnie) zostawiłbym jako node.

Ok, dodam też tego typu słówka z części piątej poradnika do słowniczka na swoim branchu. Jak zrobimy merge ostatecznej wersji części piątej, to zrobię PR z nowymi terminami. Proponuje żeby wszyscy tak robili, bo często jedna część poradnika nawiązuje do drugiej, i warto żeby terminologia się pokrywała, dlatego jeśli część nad którą pracujesz wprowadza jakieś nowe słówka, dodawaj je na swoim branchu do słowniczka, aby kazda nastepna osoba mogla z nich momentalnie korzystać

Reply:
```suggestion
Alternatywnie, możesz dodawać pola używając skrótu autouzupełnienia (<kbd>Ctrl + Space</kbd>). To pokaże wszystkie dostępne pola dla zapytań dla `File` Node.
```